common/views/components/Modal/Modal.tsx

```diff
@@ -69,7 +69,7 @@ const CloseButton = styled(Space).attrs<{ type?: string }>(props => ({
     position: absolute;
     left: 50%;
     top: 50%;
-    transform: translateX(-50%) translateY(-50%);
+    transform: translateX(-50%) translateY(-50%) scale(1.5);
```

Comment:
I don't think this is the right solution, but probably good enough for now/here? I've left a [comment on the Figma file](https://www.figma.com/design/KkVDUZ791MjNwoFEmefwdx?node-id=1192-10812&m=dev#1145953479) suggesting that we use a different svg `cross` icon that stretches the full 24x24 viewbox